src/components/Bookings/Bookings.jsx

```diff
@@ -1,17 +1,26 @@
 import Search from "@/components/Search/Search";
 import SearchResult from "../SearchResult/SearchResult";
+import { useState } from "react";
+import BookingForm from "../BookingForm/BookingForm";
+import FakeBookings from "@/data/fakeBookings.json";
 // import SearchResults from "@/componentsSearchResults.js";
-// import FakeBookings from "@/data/fakeBookings.json";
 
 const Bookings = () => {
+  const [bookings, setBookings] = useState([...FakeBookings]);
```

Comment:
Wondering why you are using ...FakeBookings? The fakebookings.json contains an array so I don't think there is any particular reason?